pulp_deb/app/serializers/content_serializers.py

```diff
@@ -273,6 +277,43 @@ def from822(cls, data, **kwargs):
                 # also save the fields not in TRANSLATION_DICT
                 args["custom_fields"][k] = v
 
+        # Delete package fields with values of incorrect type
+        if "installed_size" in args:
+            try:
+                int(args["installed_size"])
+            except (TypeError, ValueError):
+                log.warn(
+                    _(
+                        "Dropping 'Installed-Size' field from '{}', \
+                         since the value '{}' is of incorrect type."
```

Comment:
```suggestion
                        (
                            "Dropping 'Installed-Size' field from '{}', "
                            "since the value '{}' cannot be casted to type 'int'."
                        )
```
This is needed to avoid having indentation make it into the final string, your version resulted in the following warning:
```
pulp_deb.app.serializers.content_serializers:WARNING: Dropping 'Installed-Size' field from code-insiders,                          since the value NaN is of incorrect type.
```